guides/release/contributing/index.md

```diff
@@ -23,6 +23,18 @@ Have a look at the [list of main repositories](repositories) to learn about the
 
 If you need some inspiration, you can check out the [Help Wanted](https://help-wanted.emberjs.com/) dashboard to browse for issues. If you are a beginner, look out for issues with the "Help wanted" and "Good first issue" labels.
 
+# Changing the API documentation
+
+To change the [API reference guidance](https://api.emberjs.com/ember/release), find the relevant section for the *package*, *method* or *class* you wish to change.
+At the top of the section, you will find a link to the 'defined in' or 'inherited from' file.
+Following this link will open the file where the guidance and the code for the function is defined.
+
+Note, the documentation is inline with the code to help ensure the code and documentation match.
+For example, near the top of the [`store.createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord) API guidance section, you can find the 'defined in' link to the specific `.ts` file where the guidance is inline: [`ds-model-store.ts`](https://github.com/emberjs/data/blob/v3.26.0/packages/store/addon/-private/system/ds-model-store.ts).
+You can then edit the guidance within the `.ts` file.
```

Comment:
Sample screenshot:

<img width="1200" alt="defined-in-link-for-method" src="https://user-images.githubusercontent.com/16869656/122662808-3a661400-d196-11eb-8ffe-30b59337d979.png">

A possible update to the paragraph:

```suggestion
Here is an example of updating a method. At the top of the section for [`store.createRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/createRecord?anchor=createRecord), you can find the words "Defined in."

[Insert screenshot and provide alternate text if needed]

Next to the words is, once again, the link to the source code [`ds-model-store.ts`](https://github.com/emberjs/data/blob/master/packages/store/addon/-private/system/ds-model-store.ts).
```